inscriptions/ISic000327.xml

```diff
@@ -180,26 +184,42 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
             <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+                <ab>
+                    <lb n="1"/>
+                    <expan><abbr>d</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan>
+                    <lb n="2"/> <name>Aemilius</name>
+                    <lb n="3"/> <name>Priscus</name>
+                    <lb n="4"/>
+                    <expan><abbr>vix</abbr><ex>it</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>ann</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <num value="2">II</num> 
+                    <lb n="5"/>
+                    <expan><abbr>mensi</abbr><ex>bu</ex></expan>s <g ref="#interpunct">·</g> <num value="2">II</num>
+                    <lb n="6"/> 
+                    <expan><abbr>die</abbr><ex>bu</ex></expan>s <g ref="#interpunct">·</g> <num value="4">IV</num> 
+                </ab>
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="it" resp="#AR">
+                <p>Consacrato agli Dei Mani Emilio Prisco visse due anni due mesi quattro giorni</p>
+                </div>
+            <div type="translation" xml:lang="en" resp="#AR">
+                <p>Sacred to the Shades Aemilius Priscus lived two years two months four days</p>
             </div>
             <div type="commentary">
                 <p><!--commented out pending revision-->
                     <!--D.M.S. / Aemilius / Priscus / vix.ann.II / mensis II / dies IV--></p>
             </div>
             <div type="bibliography">
                 <listBibl type="edition">
+                    <bibl>
+                        <author>Carrera</author>
```

Comment:
I like the thought, but we're not adding additional bibliography yet: when we do, it needs to be fully referenced, both in terms of the specific place in the work, but also the work itself needs to be externally referenced (normally we put it in to Zotero and point to that) - but we'll come on to all of that in the next couple of weeks. For now, better to leave this out. I know the bibliography is formally incomplete, but we're treating this all as work in progress for now.